.gitignore

```diff
@@ -1,2 +1,23 @@
-.idea
```

Comment:
I don't think that this folder with files should be in the git

Reply:
didn't notice sorry